### DISCUSSION TITLE
v3 adding liquidity using the URL - query params, HELP

### DISCUSSION THREAD

**Discussion by bieber-t-beaver** [Uniswap/interface#7804] [Q&A] [answered]
What are the supported query params for adding liquidity...  

https://app.uniswap.org/positions/create/v3?currencyA={token0}&currencyB={token1}

*** instead of { token0, token1, fee } -- should the pool ID be used instead?
What are the other names to apply the fee, min price, max price, chain values?

**Comment by mikeghen** [comment]
Would also be good to add `hooks`, `tickSize` as a query param

**Comment by bieber-t-beaver** [comment, marked as answer]
Example - complete with price range and hooks

https://app.uniswap.org/positions/create/v3?&currencyA=NATIVE
&currencyB=0xaf88d065e77c8cC2239327C5EDb3A432268e5831
&chain=arbitrum
&step=1
&hook=undefined
&fee={'feeAmount':500, 'tickSpacing':1, 'isDynamic':false}
&priceRangeState={'priceInverted':false, 'fullRange':false, 
                                'minPrice':'4681.310462766008', 
                                'maxPrice':'4800.975863143462',
                                'initialPrice':''}
